libpod/pod_api.go

```diff
@@ -447,5 +449,6 @@ func (p *Pod) Inspect() (*PodInspect, error) {
 		},
 		Containers: podContainers,
 	}
+	defer p.newPodEvent(events.Inspect)
```

Comment:
Why do you call these as defer rather then just calling them?

Reply:
Agree - defer is definitely slower, and we're always calling at the end of functions anyways

Reply:
removingf as you guys dont want inspect anyways